File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusApiKeyForm.kt
package com.sdkwork.backend

data class PlusApiKeyForm(
    val name: String? = null,
    val keyValue: String? = null,
    val type: String? = null,
    val status: String? = null,
    val expireTime: String? = null,
    val description: String? = null,
    val lastUsedTime: String? = null
)

File: sdkwork-backend-sdk-csharp/Models/PlusApiKeyForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusApiKeyForm
    {
        public string? Name { get; set; }
        public string? KeyValue { get; set; }
        public string? Type { get; set; }
        public string? Status { get; set; }
        public string? ExpireTime { get; set; }
        public string? Description { get; set; }
    }
}
